mentoring/public/js/questionnaire.js

```diff
@@ -27,7 +29,11 @@ function MessageView(element, mentoring) {
                 popupDOM.css('maxHeight', data.height);
             } else {
                 popupDOM.css('height', '');
+                popupDOM.css('maxHeight', '');
             }
+            // .tip-choice-group should always be the same height as the popup
+            // for scrolling to work properly.
+            popupDOM.find('.tip-choice-group').height(popupDOM.height());
```

Comment:
@e-kolpakov @antoviaque I tried, but it's not possible as far as I an tell. When parent has a max-height and you try to use percentage height on the child, things don't work as expected, this explains why: http://stackoverflow.com/questions/5657964/css-why-doesn-t-percentage-height-work/5658062#5658062